assets/js/esDevelopersItaliaQuery.js

```diff
@@ -964,30 +966,31 @@ esDevelopersItaliaQuery.prototype.renderSoftware = function (software) {
   var category_id = this.getSoftwareType(software);  // generic-sw vs public-sw
   var description = this.languageFallback(software.publiccode.description);
 
+  const placeholder = (category_id == 'public-sw')
+    ? '/assets/images/cover_softwareriuso.png'
```

Comment:
In another PR we should really change this filename to `software_riuso` and fix all occurrences, perfect for hacktoberfest :smile:

Reply:
Agreed :wink: